### PR TITLE
mariadb, redis and forego: Fix manifests and update

### PR DESCRIPTION
**Pull request by niheaven**
Fix https://github.com/ScoopInstaller/Main/issues/2843

ffmpeg-shared, opus-tools and teleport work fine now

### CODE REVIEW

**Comment by niheaven** [comment]
The manifests are fine, and why gh action reports error?

File: bucket/mariadb.json [hunk 1]
```diff
@@ -59,8 +59,8 @@
     ],
     "persist": "data",
     "checkver": {
-        "url": "https://downloads.mariadb.org/",
-        "regex": "Download ([\\d.]+) Stable"
+        "url": "https://downloads.mariadb.org/rest-api/mariadb/all-releases/",
+        "regex": "MariaDB Server ([\\d.]+)\""
```
Comment (pratikpc) [inline bucket/mariadb.json:63]: Need to use JSONPath here

https://github.com/ScoopInstaller/Main/blob/master/bucket/haskell.json#L33

Comment (pratikpc) [reply bucket/mariadb.json:63]: Check if this works

$.releases[?(@.status=='stable')].text

This worked for me. Tested at https://jsonpath.com/ but am not sure if Scoop would support this

Comment (pratikpc) [reply bucket/mariadb.json:63]: Or $.releases[?(@.status=='stable')].release_number to get the release_number directly

Comment (niheaven) [reply bucket/mariadb.json:63]: This is most convenient approach here, the jsonpath tested above doesn't work here and I don't know why, and that's why I use regex.

Comment (niheaven) [reply bucket/mariadb.json:63]: BTW, Scoop support jsonpath regex (https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate#using-jsonpath-in-checkver), but debug is difficult.

Comment (pratikpc) [reply bucket/mariadb.json:63]: Will create a PR to fix this jsonpath issue today.

**Comment by niheaven** [comment]
Wait for https://github.com/ScoopInstaller/Scoop/pull/4522

**Comment by tech189** [comment]
With https://github.com/ScoopInstaller/Scoop/pull/4522 merged and with the develop branch merged into the base branch, is this still blocked?

**Comment by rashil2000** [comment]
develop hasn't been merged into master yet, we should do that soon

**Comment by tech189** [comment]
Oh, I saw a lot of commits from up to a year ago come through when I updated Scoop recently and assumed those were the new features/fixes from develop!

**Comment by rashil2000** [comment]
This can be merged now?

**Pull request merged**